openedx/core/djangoapps/user_api/helpers.py

```diff
@@ -56,7 +56,7 @@ def _wrapped(*args, **kwargs):
                             func_name=func.func_name,
                             args=args,
                             kwargs=kwargs,
-                            exception=repr(ex)
+                            exception=ex.developer_message if hasattr(ex, 'developer_message') else repr(ex)
```

Comment:
This is actually unrelated to the PR-- just I was hitting an exception during development and only seeing the name of it instead of the developer-message.

developer_message is a common field on the Exceptions that are raised by the APIs related to this helper method. Part of the best practices for REST APIs is to have a developer_message and possibly also a user_message.